Added Wikipedia Data Extractor Script

akhil-chagarlamudi opened the pull request:
I have added the Wikipedia data extractor script along with instructions needed to run the script.
If any changes are to be done before getting merged, kindly let me know.

akhil-chagarlamudi commented:
@fawazsullia  Can I get the hacktober label?

fawazsullia commented:
Please update the main read me according to the template.  I need to ask the repo owner about the hacktober label, will get back tomorrow. mean while do update the main project readme

fawazsullia commented:
And a small improvment suggestion is to not hard code the search term and instead make it user input based. Will open an issue on the same and you can make the changes if required

akhil-chagarlamudi commented:
Thank you for your response. I have updated the README.md and will make PR.
Also yes, I have changed the hardcoded input to user input based. 
You may assign me the issue on this change.
